Trident/offsetfinder.c

```diff
@@ -894,11 +922,13 @@ uint32_t find_setreuid(void) {
         case iPhone41_iOS932: return 0x2a985c;
         case iPhone41_iOS933: return 0x2a9988;
         case iPhone41_iOS934: return 0x2a9988;
+        case iPhone51_iOS933: return 0x2af7b8;
         case iPhone52_iOS920: return 0x2a9e24;
         case iPhone52_iOS921: return 0x2a9f34;
         case iPhone52_iOS930: return 0x2af658;
         case iPhone52_iOS932: return 0x2af5f8;
         case iPhone52_iOS933: return 0x2af7b8;
+        case iPhone52_iOS934: return 0x2af7b8;// ? to be verified
```

Comment:
You can use joker -s to find UNIX syscalls, it should be able to give you the address of setreuid